source/extensions/all_extensions.bzl

```diff
@@ -0,0 +1,9 @@
+# Return all extensions to be compiled into Envoy.
+# TODO(mattklein123): Every extension should have an independent Bazel select option that will
```

Comment:
@jmillikin-stripe do you know of any neat Bazel hacks to make optional configuration of filters easy from a user perspective and scalable? Seems that in the Bazel world, this should happen via WORKSPACE.

Reply:
Continuing this discussion at https://github.com/envoyproxy/envoy/issues/2069#issuecomment-376201653.

Reply:
Can we keep what I started here for now? I think it's a relatively easy way of getting started on conditional compilation.